### ISSUE TITLE
Calendar not working in 2023.4

### ISSUE THREAD

**Issue by lancer73** [pippyn/Home-Assistant-Sensor-Afvalbeheer#389]
When using HA core 2023.4, the sensors are working, but the calendar gives a "Could not retrieve events"

Attribute values of the calendar entity:
```
message: papier
all_day: true
start_time: '2023-04-06 00:00:00'
end_time: '2023-04-07 00:00:00'
location: ''
description: ''
icon: mdi:delete-empty
friendly_name: Afvalbeheer alphenaandenrijn
```

**Comment by pippyn** [comment]
Fixed in v5.1.1

**Comment by lancer73** [comment]
Confirmed! Thnx.
